src/blockchain/deposit_strategy/interface.py

```diff
@@ -1,17 +1,57 @@
+import logging
 from abc import ABC, abstractmethod
 
+from blockchain.deposit_strategy.gas_price_verifier import GasPriceCalculator, get_pending_base_fee
 from blockchain.typings import Web3
+from metrics.metrics import DEPOSITABLE_ETHER, POSSIBLE_DEPOSITS_AMOUNT
+from web3.types import Wei
+
+logger = logging.getLogger(__name__)
 
 
 class ModuleDepositStrategyInterface(ABC):
-    def __init__(self, w3: Web3, module_id: int):
-        self.w3 = w3
-        self.module_id = module_id
+    _DEPOSITABLE_KEYS_THRESHOLD = 0
 
-    @abstractmethod
-    def is_gas_price_ok(self) -> bool:
-        pass
+    def __init__(self, w3: Web3):
+        self._w3 = w3
+
+    def _get_possible_deposits_amount(self, module_id: int) -> int:
+        depositable_ether = self._w3.lido.lido.get_depositable_ether()
+        additional_ether = self._additional_ether()
+        if additional_ether > 0:
+            logger.info({'msg': 'Adding mellow vault balance to the depositable check', 'vault': additional_ether})
```

Comment:
This is an abstract class that can be inherited by non-mellow vaults. Let's use here abstract description.

Reply:
Removed